src/common/components/repositories-home/index.js

```diff
@@ -33,12 +36,22 @@ class RepositoriesHome extends Component {
 
   componentDidMount() {
     const { authenticated } = this.props.auth;
+    const { dispatch } = this.props;
     const owner = this.props.user.login;
 
     if (authenticated) {
-      this.props.dispatch(fetchUserSnaps(owner));
+      dispatch(fetchUserSnaps(owner));
+      if (!interval) {
+        interval = setInterval(() => {
+          dispatch(fetchUserSnaps(owner, false));
+        }, SNAP_POLL_PERIOD);
+      }
```

Comment:
Poll `fetchUserSnaps` with a fifteen second interval, only if an existing timer has not been set up.